src/dmd/objc.d

```diff
@@ -75,12 +75,12 @@ struct ObjcSelector
 
     extern (D) static ObjcSelector* lookup(const(char)* s, size_t len, size_t pcount)
```

Comment:
Since this is `extern(D)` you might be able to just change it to an array ?

Reply:
Yeah, that should be pretty easy, but I wanted to keep the changes in this one minimal. I can submit that as a followup.